docs/content/concepts/io-management/unconnected-inputs.mdx

```diff
@@ -9,16 +9,14 @@ description:
 
 | Name                                                                 | Description                                                                 |
 | -------------------------------------------------------------------- | --------------------------------------------------------------------------- |
-| <PyObject module="dagster" object="root_input_manager" decorator />  | The decorator used to define a Root Input Manager.                          |
-| <PyObject module="dagster" object="RootInputManager" />              | The base class to define how to load inputs to ops at the root of a job.    |
 | <PyObject module="dagster" object="dagster_type_loader" decorator /> | The decorator used to define a Dagster Type Loader.                         |
 | <PyObject module="dagster" object="DagsterTypeLoader" />             | The base class used to specify how to load inputs that depends on the type. |
 
 ## Overview
 
 Ops in a job may have input definitions that don't correspond to the outputs of upstream ops. You can provide values for these inputs in a few different ways. Dagster checks each, in order, and uses the first that's available:
 
-- **Root Input Manager (experimental)** - If the input to a job comes from an external source, such as a table in a database, often it makes sense to define a resource that's responsible for loading it. This makes it easy to swap out implementations in different jobs and mock it in tests. A <PyObject module="dagster" object="RootInputManager" />, which can be referenced from <PyObject module="dagster" object="In" />s, is a resource that loads unconnected inputs.
+- **Input Manager (experimental)** - If the input to a job comes from an external source, such as a table in a database, often it makes sense to define a resource that's responsible for loading it. This makes it easy to swap out implementations in different jobs and mock it in tests. A special IOManager, which can be referenced from <PyObject module="dagster" object="In" />s, can be used to loads unconnected inputs.
```

Comment:
not sure if the "input manager" concept will be experimental because it's just an IO manager which is stable